libres/lib/include/ert/python.hpp

```diff
@@ -61,6 +61,7 @@ class ILogger {
  * is called, and before Python shuts down, but is valid for all other times.
  */
 std::shared_ptr<ILogger> get_logger(const std::string &name);
+
```

Comment:
Remove the new line here I guess.

Reply:
I think the terminating `newline` here and the thereof in `metric.hpp`, shows some inconsistency when it comes to clang-format

Reply:
I could remove the newline on my computer and `clang-format` did not complain?